res_partner_operating_unit/models/res_partner.py

```diff
@@ -6,28 +6,21 @@
 
 class ResPartner(models.Model):
     _inherit = "res.partner"
+    _check_company_auto = True
 
     @api.model
-    def operating_unit_default_get(self, uid2=False):
+    def _default_operating_unit(self, uid2=False):
         if not uid2:
-            uid2 = self._uid
+            uid2 = self.env.user.id
```

Comment:
I couldn't find where this `uid2` parameter would be used.
Perhaps this parameter, along with this code block could be removed